packages/graalvm/src/main/kotlin/elide/runtime/gvm/internals/vfs/HybridVfs.kt

```diff
@@ -34,7 +35,7 @@ import elide.runtime.gvm.internals.vfs.EmbeddedGuestVFSImpl.Settings
 internal class HybridVfs private constructor(
   private val backing: FileSystem,
   private val overlay: FileSystem,
-) : FileSystem {
+) : GuestVFS {
```

Comment:
The `HybridVFS` should have always been a `GuestVFS` to begin with.

Reply:
Good catch